spec/controllers/idv/how_to_verify_controller_spec.rb

```diff
@@ -236,17 +261,6 @@
         expect(@analytics).to have_received(:track_event).with(analytics_name, analytics_args)
       end
     end
-
-    context 'undo/back' do
```

Comment:
It seems like this is duplicative in intent of https://github.com/18F/identity-idp/blob/main/spec/controllers/idv/how_to_verify_controller_spec.rb#L171